css/CSS2/selectors/first-child-selector-003.xht

```diff
@@ -5,12 +5,13 @@
   <link rel="author" title="Ian Hickson" href="mailto:ian@hixie.ch"/>
   <link rel="alternate" href="http://www.hixie.ch/tests/adhoc/css/selectors/child/001.xml" type="application/xhtml+xml"/>
   <link rel="help" href="http://www.w3.org/TR/CSS21/selector.html#first-child" />
+  <link rel="match" href="../reference/ref-this-text-should-be-green.xht" />
   <style type="text/css">
-   html { color: green; }
-   html:first-child { color: red ! important; }
+   html { color: red; }
+   html:first-child { color: green ! important; }
```

Comment:
Hmm, I wonder if the intention here was `html > *:first-child`?

Reply:
(Also I don't understand why the `!important` is there.)

Reply:
I'm not sure, going back to [the source](http://www.hixie.ch/tests/adhoc/css/selectors/child/001.xml) it's originally `root:first-child`.

Reply:
Let's assume it's a test that the root element matches `:first-child` and drop the `!important`?

Reply:
Okay, I changed it to that.